wiki/Internet_Relay_Chat/id.md

```diff
@@ -1,40 +1,41 @@
 
 ## osu!Bancho
 
-osu!Bancho menyediakan gateway untuk mengakses IRC (obrolan dalam game). Anda dapat terhubung dengan klien anda sendiri dan mulai obrolan dengan orang lain tanpa harus menggunakan klien osu!. Mohon perhatikan bahwa prokotol IRC ini telah dimodifikasi oleh karena itu jangan harap fitur klien anda berfungsi dengan benar.
+osu!Bancho menyediakan gateway untuk mengakses IRC (obrolan dalam game). Anda dapat terhubung dengan klien Anda sendiri dan mulai obrolan dengan orang lain tanpa harus menggunakan klien osu!. Mohon perhatikan bahwa prokotol IRC ini telah dimodifikasi oleh karena itu jangan harap fitur klien Anda berfungsi dengan benar.
 
-**Pemberitahuan: [HexChat](https://hexchat.github.io/) diketahui memiliki masalah dengan IRC osu!** ([laporan bug dari GitHub HexChat](https://github.com/hexchat/hexchat/issues/818)), jika hal ini mengganggu anda, pertimbangkan untuk menggunakan klien lain .
+**Pemberitahuan: [HexChat](https://hexchat.github.io/) diketahui memiliki masalah dengan IRC osu!** ([laporan bug dari GitHub HexChat](https://github.com/hexchat/hexchat/issues/818)), jika hal ini mengganggu Anda, pertimbangkan untuk menggunakan klien lain .
 
 ## Cara Menghubungkan
 
 Setelah Anda memiliki klien, Anda perlu mengatur pengaturan server.
 
 - **Server:** `irc.ppy.sh`
 - **Port:** `6667` (standar)
-- **Username:** Nama pengguna osu! anda (ganti spasi dengan garis bawah)
+- **Username:** Nama pengguna osu! Anda (ganti spasi dengan garis bawah)
 - **Password:** Dapatkan di [Otentikasi IRC](https://osu.ppy.sh/p/irc).
 
-*Kata sandi IRC anda berbeda dengan kata sandi akun osu! anda. **Jangan dibagikan kepada orang lain**.*
+*Kata sandi IRC Anda berbeda dengan kata sandi akun osu! Anda. **Jangan dibagikan kepada orang lain**.*
 
 ## Perintah Dasar IRC
 
 | Perintah | Deskripsi |
 | :-- | :-- |
-| `/join <#channel>` | Bergabung ke channel |
-| `/part <#channel>` | Meninggalkan channel |
+| `/join <#channel>` | Bergabung ke kanal |
+| `/part <#channel>` | Meninggalkan kanal |
 | `/me <action>` | Mengirim pesan aksi |
 | `/ignore <username>` | Mengabaikan pengguna (menyembunyikan pesan mereka) |
+| `/query <username>` | Membuka kanal percakapan pribadi dengan pengguna lain (ganti spasi dengan tanda garis bawah) |
+
 
 ## Menonaktifkan Pemberitahuan Masuk/Meninggalkan
 
-Mengetahui pemain yang masuk atau meninggalkan adalah hal baik, namun jika kanalnya sangat ramai seperti `#osu`, anda akan menerima pemberitahuan masuk/meninggalkan terus menerus dan tidak dapat mengikuti percakapan. Oleh karena itu, biasanya pesan-pesan ini lebih disarankan untuk disembunyikan.
+Mengetahui pemain yang masuk atau meninggalkan adalah hal baik, namun jika kanalnya sangat ramai seperti `#osu`, Anda akan menerima pemberitahuan masuk/meninggalkan terus menerus dan tidak dapat mengikuti percakapan. Oleh karena itu, biasanya pesan-pesan ini lebih disarankan untuk disembunyikan.
 
 ```
```

Comment:
from this part, the logs until line 49 should be removed since at main articles may be removed already